cli/command/system/version.go

```diff
@@ -1,34 +1,52 @@
 package system
 
 import (
+	"fmt"
 	"runtime"
+	"sort"
+	"text/template"
 	"time"
 
-	"golang.org/x/net/context"
-
 	"github.com/docker/cli/cli"
 	"github.com/docker/cli/cli/command"
 	"github.com/docker/cli/templates"
 	"github.com/docker/docker/api/types"
 	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
 )
 
-var versionTemplate = `Client:
- Version:      {{.Client.Version}}
- API version:  {{.Client.APIVersion}}{{if ne .Client.APIVersion .Client.DefaultAPIVersion}} (downgraded from {{.Client.DefaultAPIVersion}}){{end}}
- Go version:   {{.Client.GoVersion}}
- Git commit:   {{.Client.GitCommit}}
- Built:        {{.Client.BuildTime}}
- OS/Arch:      {{.Client.Os}}/{{.Client.Arch}}{{if .ServerOK}}
-
-Server:
- Version:      {{.Server.Version}}
- API version:  {{.Server.APIVersion}} (minimum version {{.Server.MinAPIVersion}})
- Go version:   {{.Server.GoVersion}}
- Git commit:   {{.Server.GitCommit}}
- Built:        {{.Server.BuildTime}}
- OS/Arch:      {{.Server.Os}}/{{.Server.Arch}}
- Experimental: {{.Server.Experimental}}{{end}}`
+var versionTemplate = `{{with .Client -}}
+Client:{{if ne .Platform.Name ""}} {{.Platform.Name}}{{end}}
+ Version:	{{.Version}}
+ API version:	{{.APIVersion}}{{if ne .APIVersion .DefaultAPIVersion}} (downgraded from {{.DefaultAPIVersion}}){{end}}
+ Go version:	{{.GoVersion}}
+ Git commit:	{{.GitCommit}}
+ Built:	{{.BuildTime}}
+ OS/Arch:	{{.Os}}/{{.Arch}}
```

Comment:
There's tabs after `Built:`  and `OS/Arch`; think that's causing the output to be not properly alligned:

```
Client:
 Version:	17.12.0-dev
 API version:	1.34 (downgraded from 1.35)
 Go version:	go1.9.2
 Git commit:	c178c6a9
 Built:	Thu Dec  7 01:01:55 2017
 OS/Arch:	linux/amd64
```

Reply:
Actually, other lines are tabs as well, but looks like there's not enough tabs after Built (perhaps spaces are better in this case)

Reply:
Discussing with @tiborvass there's a bug in tabwriter, and we can fix that after rc1

Reply:
i didn't use tabwriter because the output was even worse with it, but once I figure out what was the issue we can re-align it all. I don't think it's a big deal.